public/src/actions/users.jsx

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 import { logoutUser } from './login';
-import { SET_USER_ACCOUNT_INFO } from '../reducers/users';
+import { SET_USER_ACCOUNT_INFO, SET_USER_CALL_SID } from '../reducers/users';
```

Comment:
We don't need to keep track of the user call SID in the client.

Reply:
I am using it as a check that the user is connected to the calling system, to conditionally render.